UI/NativeApp.cpp

```diff
@@ -751,27 +754,34 @@ void DrawDownloadsOverlay(UIContext &dc) {
 		0xFF777777,
 	};
 
-	dc.Begin();
+	dc->Begin();
 	int h = 5;
 	for (size_t i = 0; i < progress.size(); i++) {
-		float barWidth = 10 + (dc.GetBounds().w - 10) * progress[i];
+		float barWidth = 10 + (dc->GetBounds().w - 10) * progress[i];
 		Bounds bounds(0, h * i, barWidth, h);
 		UI::Drawable solid(colors[i & 3]);
-		dc.FillRect(solid, bounds);
+		dc->FillRect(solid, bounds);
+	}
+	dc->End();
+	dc->Flush();
+
+	if (g_TakeScreenshot) {
+		TakeScreenshot();
```

Comment:
We need to capture the buffer as part of submitting the frame's commandbuffers. Well technically I guess we can just submit more command buffers... Hm. Well, might revisit later.